armi/operators/operator.py

```diff
@@ -590,21 +595,30 @@ def addInterface(
             )
 
         iFunc = self.getInterface(function=interface.function)
+
         if iFunc:
-            raise RuntimeError(
-                "Cannot add {0}; the {1} already is designated "
-                "as the {2} interface. Multiple interfaces of the same "
-                "function is not supported.".format(
-                    interface, iFunc, interface.function
+            if issubclass(type(iFunc), type(interface)):
+                runLog.info(
+                    "Ignoring Interface {newFunc} because existing interface {old} already "
+                    " more specific".format(newFunc=interface, old=iFunc)
+                )
+                return
+            elif issubclass(type(interface), type(iFunc)):
+                self.removeInterface(iFunc)
```

Comment:
Which of these branches gets taken is I think just a matter of the order in which a fist full of Interfaces are added. So I think it makes sense to report something to the user either way. Could you add a `runLog` message saying that we are replacing the existing interface with the new one?